project/eScavenge/views.py

```diff
@@ -70,3 +71,64 @@ def teamPage(request):
     teamhistory = userpage.history.all()
     context = {'team': userpage, 'teamlist': teamlist, 'teamhistory': teamhistory, 'command': command}
     return render(request, 'teamPage.html', context)
+
+def gamemakerPage(request):
+    username = request.session.get('username')
+    message = ''
+    cli = CLI(COMMANDS)
+    if username != GM.username:
+        return HttpResponseForbidden()
+    if (request.method == 'POST'):
+        if (request.POST.get("selectGame")):
+            request.session['curgame'] = request.POST["gameObject"]
+    if('curgame' in request.session.keys()):
+        curgame = request.session.get('curgame')
+        try:
+            gameteam = Game.objects.get(name = curgame).teams.all()
+        except:
+            gameteam = []
+            message = "No Teams Yet!"
+        try: 
+            gamemarks = Game.objects.get(name = curgame).landmarks.all()
+        except:
+            gamemarks = []
+            message = message + " No Landmarks Yet!"
+        gamestatus = "NOT WORKING YET"
+        if (Game.objects.get(name = curgame).started is False and Game.objects.get(name = curgame).ended is False):
```

Comment:
this whole if block is kinda ugly. You're making 6 database calls when you only need one. Get the database object in a variable and use that for all these, it will also make the code easier to read. Oh, and started and ended are boolean, so you can just use `if not mygame.started and not mygame.ended`